spec/requests/political_edit_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe "Documents" do
+  describe "GET /documents/:document/political" do
+    context "without managing editor permission" do
+      it "returns a forbidden status" do
+        @edition = create(:edition)
+        login_as(create(:user))
+        get political_path(@edition.document)
+        expect(response.status).to eq(403)
+      end
+    end
+
+    context "with managing editor permission" do
+      it "does not return a forbidden status" do
+        @edition = create(:edition)
+        login_as(create(:user, :managing_editor))
+        get political_path(@edition.document)
+        expect(response.status).to eq(200)
+      end
+    end
```

Comment:
Might not these two to be in context blocks since that can probably be captured in the it description. 

We might want to have the POST endpoint covered too.

Also do we want to check the forbidden one shows the expected text?